soc/arm/st_stm32/stm32f0/Kconfig.defconfig.series

```diff
@@ -15,4 +15,7 @@ config SOC_SERIES
 config SRAM_VECTOR_TABLE
 	default y
 
+config TASK_WDT_HW_FALLBACK_DELAY
+	default 100
```

Comment:
Need to add `depends on TASK_WDT_HW_FALLBACK`

Reply:
adding 
```
# adjust the fallback because of the LSI oscaillator characteristics
config TASK_WDT_HW_FALLBACK_DELAY
	depends on TASK_WDT_HW_FALLBACK
```